src/AppBundle/Resources/translations/messages.es.yml

```diff
@@ -132,6 +132,7 @@ decks:
     labelswitch: Click para cambiar a esta etiqueta. Mayús+Click para alternar esta etiqueta.
     download:
       text: Fichero de texto
+      text_cycleorder: Fichero de texto (ordenado por fecha de lanzamiento)
       octgn: Fichero OCTGN
```

Comment:
please add translation for `text_cycleorder_deluxe_after` as well.

Reply:
Solved in ad499ad72551c40d32e3ece4c9079165b524c108